pyEpiabm/pyEpiabm/sweep/spatial_sweep.py

```diff
@@ -205,10 +207,14 @@ def find_infectees_Covidsim(self, infector: Person,
             # Sample at random from the infectee cell to find
             # an infectee
             infectee = random.sample(infectee_cell.persons, 1)[0]
-            infection_distance = (DistanceFunctions.dist(
-                infector.microcell.cell.location, infectee_cell.location) /
+            # Covidsim tested each infection event by testing the ratio
+            # of the spatial kernel applied to the distance between people
+            # to the spatial kernel of the shorted distance between
+            # their cells.
+            infection_distance = (SK.weighting(DistanceFunctions.dist(
+                infector.microcell.cell.location, infectee_cell.location)) /
                 Parameters.instance().infection_radius)
-            if (infection_distance < random.random()):
+            if (infection_distance > random.random()):
```

Comment:
Just to check, is this the right way round - seems to be adding people when the infection distance is large?

Reply:
Yes bc the kernel is monotonic decreasing now. Renamed vriables to make this clear